### PR TITLE
change window peak normaliztion to bool

### PR DESCRIPTION
**Pull request by trishorts**

### CODE REVIEW

File: MassSpectrometry/MsDataFile.cs [hunk 1]
```diff
@@ -183,14 +182,15 @@ public static void WindowModeHelper(ref double[] intensities, ref double[] mArra
...
+                if (filteringParams.NormalizePeaksAccrossAllWindows)
                 {
                     double max = tempIntList.Max();
                     if (max == 0)
                     {
                         max = 1;
                     }
-                    tempIntList = tempIntList.Select(x => x / max * filteringParams.WindowMaxNormalizationValue.Value).ToList();
+                    //currently normalizing all peaks to 100
+                    tempIntList = tempIntList.Select(x => x / max * 100d).ToList();
```
Comment (zrolfs) [inline MassSpectrometry/MsDataFile.cs]: Remove the 100 and just normalize it to the maximum peak being 1 (x => x / max).

Comment (trishorts) [reply MassSpectrometry/MsDataFile.cs]: done

File: MassSpectrometry/MsDataFile.cs [hunk 2]
```diff
@@ -183,14 +182,14 @@ public static void WindowModeHelper(ref double[] intensities, ref double[] mArra
                     tempIntList.Add(intensities[arrayIndex]);
                     tempMzList.Add(mArray[arrayIndex]);
                 }
-                if (filteringParams.WindowMaxNormalizationValue.HasValue)
+                if (filteringParams.NormalizePeaksAccrossAllWindows)
```
Comment (zrolfs) [inline MassSpectrometry/MsDataFile.cs]: Across? Accross?

Comment (zrolfs) [reply MassSpectrometry/MsDataFile.cs]: Just this, then I'll accept

**Pull request merged**